frontend/src/components/MyApplicationItem/MyMissionItem/__tests__/useJudgment.test.ts

```diff
@@ -0,0 +1,155 @@
+import { renderHook, act } from "@testing-library/react";
+import useMissionJudgment from "../useMissionJudgment";
+import { postMyMissionJudgment } from "../../../../api";
+import useTokenContext from "../../../../hooks/useTokenContext";
+import { isJudgmentTimedOut } from "../../../../utils/validation/judgmentTime";
+import { MISSION_STATUS } from "../../../../constants/recruitment";
+import { JUDGMENT_STATUS } from "../../../../constants/judgment";
+import { Mission } from "../../../../../types/domains/recruitments";
+
+jest.mock("../../../../api");
+jest.mock("../../../../hooks/useTokenContext");
+jest.mock("../../../../utils/validation/judgmentTime");
+
+describe("useMissionJudgment 훅 테스트", () => {
+  const mockRecruitmentId = 1;
+  const mockMissionItem: Mission = {
+    id: 1,
+    title: "테스트 미션",
+    description: "테스트 설명",
+    submittable: true,
+    submitted: true,
+    startDateTime: "2023-01-01T00:00:00",
+    endDateTime: "2023-12-31T23:59:59",
+    status: MISSION_STATUS.SUBMITTING,
+    runnable: true,
+    judgment: {
+      pullRequestUrl: "https://github.com/test/pr",
+      commitHash: "abcdef1234567890",
+      status: JUDGMENT_STATUS.STARTED,
+      passCount: 0,
+      totalCount: 10,
+      message: "",
+      startedDateTime: "2023-06-01T12:00:00",
+      commitUrl: "https://github.com/test/commit",
+    },
+  };
```

Comment:
a. 객체 구조가 꽤 복잡해져서, 테스트 코드에서 미션 객체를 직접 수정할 때 실수할 확률도 있을것 같아요! 혹시 이걸 팩토리 함수로 만들어서 사용해보면 어떨까요~?

```suggestion
  function createMockMission(overrides = {}) {
  return {
    ...mockMissionItem,
    ...overrides,
  };
}

// 사용 예:
it('제출되지 않은 미션은 판정 불가능해야 한다', () => {
  const unsubmittedMission = createMockMission({ submitted: false });
  // ... 테스트 로직
});
```

Reply:
제안 감사합니다! 일괄 반영하겠습니다.